app/views/admin/workshops/_form.html.haml

```diff
@@ -1,4 +1,4 @@
-= simple_form_for [:admin, @workshop], url: :admin_workshops do |f|
+= simple_form_for [:admin, @workshop], html: { novalidate: true } do |f|
```

Comment:
I haven't looked at this yet but I did a quick search for `novalidate` so my first question when I look is why novalidate isn't used in any other forms?

Reply:
that is a good point argument but I've looked at the history of the two files and they were created almost identical in the same commit https://github.com/codebar/planner/commit/7ef5049753b1bafe324a6d18ca8765c12196a10e and they had a fairly parallel edit history so I'm confident there should be no problem with this PR